data-prepper-plugins/aws-lambda/build.gradle

```diff
@@ -26,7 +26,12 @@ dependencies {
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
     testImplementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
     testImplementation project(':data-prepper-test-common')
-    testImplementation project(':data-prepper-plugins:parse-json-processor')
+    testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
+    testImplementation 'org.powermock:powermock-api-mockito2:2.0.9'
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
You shouldn't need any of these four lines. They are provided by the root project.